<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CN_HW8</class>
 <widget class="QMainWindow" name="CN_HW8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CN_HW8</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileDialog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>680</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="drawGraphics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="drawGraphics2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>591</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
